sycl/test-e2e/lit.cfg.py

```diff
@@ -15,6 +15,23 @@
 from lit.llvm.subst import ToolSubst, FindTool
 
 # Configuration file for the 'lit' test runner.
+config.backend_to_target = {
+    "level_zero": "target-spir",
+    "opencl": "target-spir",
+    "cuda": "target-nvidia",
+    "hip": "target-amd",
+    "native_cpu": "target-native_cpu",
+}
+config.target_to_triple = {
+    "target-spir": "spir64",
+    "target-nvidia":"nvptx64-nvidia-cuda",
+    "target-amd":"amdgcn-amd-amdhsa",
+    "target-native_cpu":"native_cpu",
+}
+config.triple_to_target = {v:k for k,v in config.target_to_triple.items()}
```

Comment:
What's the difference between a "target" and the "triple" it corresponds to, given there's a 1:1 mapping between them? Is target just a short name for the triple? Is it an alias to allow changing the target triples without updating all the tests that refer to it?

Reply:
Short name was part of the motivation (at least for me). Also, Nick mentioned that there is some work to have two different spir triples (spir64 and spirv<something>, I think).